index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>freeCodCamp Survey Form</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+<br>
+<form action="" id="Survey-from"></form>
+<label for="name">Name</label>

```

Comment:
- [ ] I think you should wrap the form content between the `<form>` tags  🧐👀
> to make the form works correctly 
```html
<form action="" id="Survery-form">
    <!-- form content here -->
</form>
```
- [ ] It would be better if you wrap your `<form>` tag between the semantic tag `<section>` 👍
> It's always better to use HTML5 semantic tags
```html
<section> 
   <form>
     ...
   </form>
</section>
```